prow/cmd/sinker/main.go

```diff
@@ -151,6 +159,44 @@ type sinkerReconciliationMetrics struct {
 	podRemovalErrors map[string]int
 }
 
+// Prometheus Metrics
+var (
+	sinkerMetrics = struct {
+		podsCreated      prometheus.Gauge
+		timeUsed         prometheus.Gauge
+		podsRemoved      *prometheus.GaugeVec
+		podRemovalErrors *prometheus.GaugeVec
+	}{
+		podsCreated: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "pods_created",
+			Help: "Number of pods created in each sinker cleaning.",
+		}),
+		timeUsed: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "loop_duration_nanoseconds",
```

Comment:
Looking at our current deployment, the durations are, in milliseconds:

```
09904
38410
32159
26525
33941
19871
24093
31511
25541
24136
29299
24643
40042
27094
28356
52045
47983
32091
38641
58449
39267
38515
25763
29176
21509
16395
14841
15243
07849
09837
12683
06597
07081
16407
24875
27333
14500
09266
31573
50455
46383
45220
51208
```

We could probably display this metric in milliseconds or seconds, nanosecond precision not super useful here

Reply:
Done.